docs-mslearn/framework/understand/ingestion.md

```diff
@@ -108,6 +108,21 @@ At this point, you have a data pipeline and are ingesting data into a central da
 
 <br>
 
+## Data Ingestion Metrics
+
+| **Category** | **Definition** | **KPI** |
+|----------|-----------|-----|
+| Data Completeness | Measures the extent to which all required data fields are present in the dataset and tracks the overall data completeness trend over a specified period.| Percentage of data fields that are complete and the overall data completeness over time. |
+| Data Quality Performance | Measures the percentage of successful data quality checks and the total number of data quality checks conducted within a specified period. | Number of data quality checks conducted and the percentage of successful data quality checks . |
+|Data Ingestion Frequency | Measures how often data is ingested into the system. | Number of data ingestion events per unit of time (daily, weekly, monthly, quarterly, annually). |
+| Volume of Data Ingested | Measures the total volume of data ingested into the repository. | Total volume of data ingested into the repository.  |
+| Growth Rate | Measure the rate at which the volume of data ingested is increasing over time. | Percentage increase of total data volume in repository per unit of time. |
+| Ingestion Latency | Measures the average time taken for data to be ingested into the repository and tracks the trend of this latency over a specified period. | Mean time of data ingestion latency and the latency trend over a specified period. |
+| Historical Data Availability | Measures the lookback period of data that is ingested and available for analysis. | Span of historical data ingested. |
```

Comment:
This name needs some work, but I do like it. I've thought about this one as well. We need to know what data is missing so we can backfill it. Should this be bound to months with complete data over the retention/reporting period?

Reply:
Maybe along the lines of "Historical data span"? (Other options include, Data lookback period, Data retention period
, Ingested data horizon, Data history range) 

I think so because that'll ensure the KPI reflects the availability of fully usable historical data, providing a more accurate measure of data completeness and reliability over time. How do you suggest I modify?